crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -167,6 +167,8 @@ type PendingForkchoiceUpdate<PayloadAttributes> =
 /// # Panics
 ///
 /// If the future is polled more than once. Leads to undefined state.
+///
+/// Note: soon deprecated. See `reth_engine_service::EngineService`.
```

Comment:
```suggestion
/// NOTE: soon deprecated. See `reth_engine_service::EngineService`.
```

can we also add a deprecation attribute?
https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute